### PR TITLE
added sections on continuous integration and configuration management

### PR DESCRIPTION
**Pull request by echohack**
doggo
Signed-off-by: echohack <echohack@users.noreply.github.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/docs/habitat-and-other-software.html.md.erb [hunk 1]
```diff
@@ -34,3 +36,25 @@ Kubernetes is a portable, extensible open-source platform for managing container
...
+
+Your continuous integration server can promote a Habitat package (a .hart file) to a channel by calling the Habitat CLI. This promotion method allows you to deploy a new version of your application in a pull-based manner using the Habitat Supervisor. The Habitat Supervisor can upgrade the application itself, including downloading and installing new binaries, starting the service in the correct manner, managing application choreography, and managing configuration files. When it is time to deploy your application to a production environment, you will want to consider using a scheduling tool or provisioning tool to help manage infrastructure resources in addition to promoting Habitat packages.
+
+## <a name="habitat-and-configuration-management" id="habitat-and-configuration-management" data-magellan-target="habitat-and-configuration-management">Habitat and Configuration Management</a>
+#### Examples: [Ansible](https://www.ansible.com/), [Chef](https://www.chef.io/chef/), and Puppet(https://puppet.com/)
```
Comment (cnunciato) [inline www/source/docs/habitat-and-other-software.html.md.erb]: Needs some `[]`s around `Puppet`.

**Comment by cnunciato** [comment]
@echohack It looks like this branched from a commit that isn't in `master`, and I don't see any other open PRs from you. Did you want to submit this new content as a single PR?

**Comment by echohack** [comment]
@cnunciato Hey Chris, not trying to merge this into master for now. I'm collaborating with @ryankeairns on this one. Let me get some feedback from him first and then we'll work on getting it merged to master.

**Comment by echohack** [comment]
Went ahead and pushed this to master in #5309

**Pull request closed without merging**